requirements.txt

```diff
@@ -3,3 +3,4 @@ django-model-utils>=4.0
 jsonfield>=3.1.0,<4.0.0
 cryptography>=2.4.0,<2.10.0
 pyopenssl>=17.5.0,<20.0.0
+swapper~=1.1.0
```

Comment:
I think this should be `swapper~=1.1`
Pay attention: https://github.com/openwisp/openwisp-ipam/commit/af5ad605f7c0d5efd7b3343b2e810cdc15896c0a
Let's make sure all dependencies are pinned

Reply:
`swapper~=1.1.0` == `swapper>=1.1,swapper<1.2`
While,
`swapper~=1.1` == `swapper>=1.1,swapper<2`

Wouldn't having upper limit as `swapper<2` be very lenient given swapper updates are infrequent and not documented?

Reply:
Thanks for clarifying, I think we can relax the requirement because this module is pretty stable.